de.metas.adempiere.adempiere/base/src/main/java/de/metas/user/UserId.java

```diff
@@ -124,4 +124,10 @@ public boolean isRegularUser()
 	{
 		return !isSystemUser();
 	}
+
+	@Nullable
+	public static UserId ofRepoIdOrNullISystem(final int repoId)
```

Comment:
sry to bother you again with this, but there is an `f` missing 😊

Reply:
actually that was not a typo, I was thinking `ofRepoIdOrNullIgnoreSystem`, but now that u mentioned it, `ofRepoIdOrNullIfSystem` it's better 👍